pax-manifest/src/lib.rs

```diff
@@ -155,7 +155,7 @@ pub struct ComponentDefinition {
     pub primitive_instance_import_path: Option<String>,
     pub template: Option<HashMap<usize, TemplateNodeDefinition>>,
     pub settings: Option<Vec<SettingsBlockElement>>,
-    pub handlers: Option<Vec<HandlersBlockElement>>,
+    pub handlers: Option<Vec<HandlerBindingElement>>,
```

Comment:
I think given this new design, it would make more sense to move handlers into settings by adding a new enum type to SettingsBlockElement. This would both preserve user order (if we decide to serialize in a specific way we still can) and also allow us to respect comments on handlers. The current design forces comments on handlers to be placed in settings and thus erroneously re-ordered on serialization.